src/encoded/static/components/facets/defaults.js

```diff
@@ -174,20 +187,47 @@ export const DefaultExistsBinaryFacet = ({ facet, relevantFilters, queryString,
     const isFieldPartOfSearchQuery = !!query.getKeyValuesIfPresent(facet.field)[0];
 
     return (
-        facet.total > 0 || isFieldPartOfSearchQuery ?
+        facet.total > 0 || isFieldPartOfSearchQuery || forceDisplay ?
             <fieldset className="facet">
-                <legend>{facet.title}</legend>
-                <div className="facet__content--exists">
-                    {terms.map((term) => (
-                        <div key={term.key} className="facet__radio">
-                            <input type="radio" name={facet.field} value={term.key} id={term.key} checked={currentOption === term.key} onChange={handleRadioClick} />
-                            <label htmlFor={term.key}>
-                                <div className="facet__radio-label">{term.key}</div>
-                                <div className="facet__radio-count">{term.doc_count}</div>
-                            </label>
-                        </div>
-                    ))}
-                </div>
+                <button
+                    className="facet-expander"
+                    type="button"
+                    id={`facet-expander-${facet.field}`}
+                    aria-label={facet.field}
+                    aria-pressed={isExpanded}
+                    onClick={(e) => handleExpanderClick(e, isExpanded, facet.field)}
+                    onKeyDown={(e) => handleKeyDown(e, isExpanded, facet.field)}
+                >
+                    <legend>{facet.title}</legend>
+                    {isExpandable && <div className="facet-chevron">{svgIcon(isExpanded ? 'chevronUp' : 'chevronDown')}</div>}
+                </button>
+                <AnimatePresence>
+                    {isExpanded && (
+                        <motion.div
+                            className="facet-content"
+                            initial="close"
+                            animate="open"
+                            exit="close"
+                            transition={{ duration: 0.2, ease: 'easeInOut' }}
```

Comment:
I think maybe this 0.2 duration should be set as a variable at the top.

Reply:
Oh, that's a good idea. I moved the whole object to the top so the ease can be tweaked too.